exercises/2-attack-of-the-pipelines/README.md

```diff
@@ -86,10 +86,10 @@ _____
 
 The Todolist application is a monorepo which has both front end and server layers in a single repo.
 
-1. Git clone the `todolist` project to the `do500-workspace` folder and checkout the `develop` branch using the following commands.
+1. Git clone the `todolist` project to the `projects` folder and checkout the `develop` branch using the following commands.
```

Comment:
Open issue #392 to track changes from `do500-workspace` to `projects` across all exercises